CONFIGURATION.md

```diff
@@ -3,7 +3,7 @@
 
 Property                                 | C/P | Range           |       Default | Description              
 -----------------------------------------|-----|-----------------|--------------:|--------------------------
-builtin.features                         |  *  |                 | gzip, snappy, ssl, sasl, regex, lz4, sasl_gssapi, sasl_plain, sasl_scram, plugins | Indicates the builtin features for this build of librdkafka. An application can either query this value or attempt to set it with its list of required features to check for library support. <br>*Type: CSV flags*
```

Comment:
remove this file from the PR.

Reply:
sorry, that was obviously a mistake, the way it gets regenerated by the build system and can get added to a commit with something like git commit -a is not great.